tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35,36,37,38},pylint{27,36,37,38}
+envlist = pylint{27,38}
```

Comment:
shouldn't need to specify any tox environments other than these non-default `pylint` environments that are still being used